src/LibraryUtilities.ts

```diff
@@ -825,7 +784,7 @@ export function findAndExpandItemByPath(pathToItem: ItemData[], allItems: ItemDa
     );
 
     if (pathToItem.length == 1) {
-        return item ? true : false;
+        return !!item;
```

Comment:
could you explain why` item ? true : false; `is the same than` !!item;`?
I know that "!" is the logical not operator but not sure why is used two times?

Reply:
@RobertGlobant20 !! operator returns the boolean value. [Ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT#double_not_!!)